operator/controllers/cleanup.go

```diff
@@ -0,0 +1,74 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	flowv1beta1 "github.com/banzaicloud/logging-operator/pkg/sdk/api/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+func (r *FlowTestReconciler) cleanUpResources(ctx context.Context, flowTestName string) error {
+	logger := log.FromContext(ctx)
+
+	matchingLabels := &client.MatchingLabels{"loggingplumber.isala.me/flowtest": flowTestName}
+
+	var podList v1.PodList
+	if err := r.List(ctx, &podList, matchingLabels); client.IgnoreNotFound(err) != nil {
```

Comment:
The k8s api support deleting multiple pods with a single request.  Is the functionality exposed with the reconciler?

Reply:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/ see delete collection

Reply:
yes, but go client was tricky to understand there, is it ok if I put todo and address this in the future?